edge/msghub/cpu2msghub/horizon/service.definition.json

```diff
@@ -27,13 +27,13 @@
             "name": "MSGHUB_API_KEY",
             "label": "The API key to use when sending messages to your instance of IBM Message Hub",
             "type": "string",
-            "defaultValue": ""
+            "defaultValue": "$MSGHUB_API_KEY"
```

Comment:
Where is the `$MSGHUB_API_KEY` variable coming from? This is information that is only known by the node owner or the organization, which is why the default value was left blank.

Reply:
I think in moving some of those variables over I copied and pasted these lines and probably forgot to remove it.